models/articles.model.js

```diff
@@ -72,3 +72,22 @@ exports.updateArticleById = (inc_votes, article_id) => {
       }
     });
 };
+
+exports.createCommentByArticleId = (article_id, postComment) => {
+  const { username, body } = postComment;
+
+  return db
+    .query(
+      `INSERT into comments (article_id, body, author)
+      VALUES ($1, $2, $3)
+      RETURNING *;`,
+      [article_id, body, username]
+    )
+    .then((result) => {
+      if (!result.rows.length) {
```

Comment:
I'm not sure that this if block is doing anything, try removing it and running the tests again. IIRC all of the errors on this endpoint will be thrown by psql.